applications/serial_lte_modem/doc/GNSS_AT_commands.rst

```diff
@@ -682,11 +682,13 @@ Delete GNSS data
 
 The ``#XGPSDEL`` command deletes GNSS data from non-volatile memory.
 This command should be issued when GNSS is activated but not started yet.
+Note! This is considered a debug feature, and is not supposed to be used in production code.
 
 Set command
 -----------
 
 The set command allows you to delete old GNSS data.
+The usage of the command does not trigger A-GPS request event and thus may delay the full functionality of A-GPS and P-GPS until the next periodic A-GPS request has been gotten.
```

Comment:
Can you split this to multiple sentences?

Reply:
Fixed